src/modules/mavlink/mavlink_main.cpp

```diff
@@ -3015,6 +3023,8 @@ Start mavlink on UDP port 14556 and enable the HIGHRES_IMU message with 50Hz:
 #endif
 	PRINT_MODULE_USAGE_PARAM_STRING('m', "normal", "custom|camera|onboard|osd|magic|config|iridium|minimal",
 					"Mode: sets default streams and rates", true);
+	PRINT_MODULE_USAGE_PARAM_STRING('n', "SoftAp", "SoftAp|wlan|name of custom wifi interface",
```

Comment:
Can you update this as well to:
`PRINT_MODULE_USAGE_PARAM_STRING('n', nullptr, "<interface_name>", "wifi/ethernet interface name", true); `

Reply:
Sorry I missed it. Updated.